peachjam/templates/peachjam/judgment_detail.html

```diff
@@ -47,4 +47,17 @@
       </dd>
     {% endif %}
   {% endwith %}
+  {% with document.attorneys.all as attorneys %}
+    {% if document.attorneys %}
```

Comment:
You've fetched the data from the db in the variable attorneys in the line above. Don't go back and check in the db again with this line. Just use the attorneys variable.